package.json

```diff
@@ -46,9 +46,11 @@
   "devDependencies": {
     "del": "^3.0.0",
     "dotenv": "^8.2.0",
+    "flat-map": "^1.0.0",
```

Comment:
Is this related to `gulp-svg-sprite` or something else?

Reply:
Nope, this should be removed.